app/containers/TextInput/FormTextInput.tsx

```diff
@@ -18,7 +18,6 @@ const styles = StyleSheet.create({
 		marginBottom: 10
 	},
 	label: {
-		marginBottom: 10,
 		fontSize: 14,
```

Comment:
Size 16 and line height 22 on Figma, right?

Reply:
Should we update the styles of this reusable component to match Figma, as this component is also used at many other places? 🤔

Reply:
Yes